src/server/lib/idapi/auth.ts

```diff
@@ -23,6 +23,8 @@ const handleError = ({ error, status = 500 }: IDAPIError) => {
     const err = error.errors[0];
     const { message } = err;
 
+    logger.error(message);
```

Comment:
it would be useful to log the stack trace

Reply:
I'm gonna make a separate PR to refactor all the errors, we've been using them wrong the whole time:

From https://github.com/guardian/gateway/blob/496851c636f1d1b17f9ea341fcc78b9dfce973fb/src/server/lib/logger.ts#L14 we can see that the first parameter is a string message, and the second is the error, which we can use to get the stack trace in the line below that.

However we've just been passing the error as the first parameter instead, which means that it doesn't provide the stack.